capa/main.py

```diff
@@ -366,24 +367,24 @@ def pbar(s, *args, **kwargs):
     return matches, meta
 
 
-def find_thread_capabilities(
```

Comment:
i wonder if we've reached the point where we should separate out this matching logic into its own namespace, and then we can have subnamespaces for static and dynamic flavors. there's hundreds of lines of logic here, which seems excessive for the main script. we can address this in another PR.

Reply:
#1684